src/App/init.sql

```diff
@@ -0,0 +1,262 @@
+
+
+--
+-- Data for Name: Bike_Parts; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public."Bike_Parts" ("PartID", "PartName", "BikeTypeID", "Stock") FROM stdin;
+1	Brake	1	50
+2	Handlebar	1	20
+3	Tire	1	100
+4	Brake	2	40
+5	Tire	2	80
+\.
+
+
+--
+-- Data for Name: Bike_Parts_Stock; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public."Bike_Parts_Stock" ("StockID", "BikeTypeID", "PartID", "Stock") FROM stdin;
+\.
+
+
+--
+-- Data for Name: Bike_Types; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public."Bike_Types" ("BikeTypeID", "BikeType") FROM stdin;
+1	Mountain Bike
+2	Road Bike
+3	Hybrid Bike
+\.
+
+
+--
+-- Name: Bike_Parts_PartID_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
+--
+
+SELECT pg_catalog.setval('public."Bike_Parts_PartID_seq"', 5, true);
+
+
+--
+-- Name: Bike_Parts_Stock_StockID_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
+--
+
+SELECT pg_catalog.setval('public."Bike_Parts_Stock_StockID_seq"', 1, false);
+
+
+--
+-- Name: Bike_Types_BikeTypeID_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
+--
+
+SELECT pg_catalog.setval('public."Bike_Types_BikeTypeID_seq"', 3, true);
+
+
+--
+-- Name: Bike_Parts_Stock Bike_Parts_Stock_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."Bike_Parts_Stock"
+    ADD CONSTRAINT "Bike_Parts_Stock_pkey" PRIMARY KEY ("StockID");
+
+
+--
+-- Name: Bike_Parts Bike_Parts_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."Bike_Parts"
+    ADD CONSTRAINT "Bike_Parts_pkey" PRIMARY KEY ("PartID");
+
+
+--
+-- Name: Bike_Types Bike_Types_BikeType_key; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."Bike_Types"
+    ADD CONSTRAINT "Bike_Types_BikeType_key" UNIQUE ("BikeType");
+
+
+--
+-- Name: Bike_Types Bike_Types_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."Bike_Types"
+    ADD CONSTRAINT "Bike_Types_pkey" PRIMARY KEY ("BikeTypeID");
+
+
+--
+-- Name: Bike_Parts Bike_Parts_BikeTypeID_fkey; Type: FK CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."Bike_Parts"
+    ADD CONSTRAINT "Bike_Parts_BikeTypeID_fkey" FOREIGN KEY ("BikeTypeID") REFERENCES public."Bike_Types"("BikeTypeID");
+
+
+--
+-- Name: Bike_Parts_Stock Bike_Parts_Stock_BikeTypeID_fkey; Type: FK CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."Bike_Parts_Stock"
+    ADD CONSTRAINT "Bike_Parts_Stock_BikeTypeID_fkey" FOREIGN KEY ("BikeTypeID") REFERENCES public."Bike_Types"("BikeTypeID");
+
+
+--
+-- Name: Bike_Parts_Stock Bike_Parts_Stock_PartID_fkey; Type: FK CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public."Bike_Parts_Stock"
+    ADD CONSTRAINT "Bike_Parts_Stock_PartID_fkey" FOREIGN KEY ("PartID") REFERENCES public."Bike_Parts"("PartID");
+
+
+--
+-- PostgreSQL database dump complete
+--
```

Comment:
Why not just do all these definitions on the create table functions, adds a lot of complexity and debug issues and technical debt. Foreign and  Primary keys should be defined on the table definition at least. Are you sure that the documentation or tool you used as reference is aware it is Postgresql. Since it follows none of those guidelines